docs/generate.jl

```diff
@@ -2,20 +2,21 @@
 import Literate
 
 MODELS = [
-    joinpath(@__DIR__, "..", "models", "Non-linear Cart-Pole"),
     joinpath(@__DIR__, "..", "models", "ACC"),
+    joinpath(@__DIR__, "..", "models", "Sherlock-Benchmark-9-TORA"),
+    joinpath(@__DIR__, "..", "models", "Sherlock-Benchmark-10-Unicycle"),
+    joinpath(@__DIR__, "..", "models", "VertCAS"),
+    joinpath(@__DIR__, "..", "models", "Single-Pendulum"),
+    joinpath(@__DIR__, "..", "models", "Double-Pendulum"),
+    joinpath(@__DIR__, "..", "models", "Airplane"),
     joinpath(@__DIR__, "..", "models", "Sherlock-Benchmark-7"),
-    joinpath(@__DIR__, "..", "models", "Sherlock-Benchmark-9"),
-    joinpath(@__DIR__, "..", "models", "Sherlock-Benchmark-10"),
-#     joinpath(@__DIR__, "..", "models", "Airplane"),
-#     joinpath(@__DIR__, "..", "models", "SinglePendulum"),
-    joinpath(@__DIR__, "..", "models", "Double Pendulum"),
-    joinpath(@__DIR__, "..", "models", "VertCAS")
+    joinpath(@__DIR__, "..", "models", "Cart-Pole")
 ]
 GENERATEDDIR = joinpath(@__DIR__, "src", "models")
 mkpath(GENERATEDDIR)
+
 for model in MODELS
-    # copy over .nnet files
+    # copy over .nnet files << TODO still needed?
```

Comment:
Do you want to fix that here?

Reply:
i wasn't sure if we can remove it. i suppose that the answer is yes (store the `.nnet` files in the `models` folder and use `@relpath`